core/src/main/java/io/grpc/transport/AbstractStream.java

```diff
@@ -133,6 +129,15 @@ public void deliverFrame(WritableBuffer frame, boolean endOfStream, boolean flus
   }
 
   /**
+   * Override this method to provide a stream listener.
+   *
+   * <p>For Override only.  Outside code should not call this method.
+   */
+  protected StreamListener listener() {
```

Comment:
I noticed that the stream listener could legally be null in some places, although now that I look through it, this class requires it to work.  

Reverted.